mod-kb-ebsco-java/mod-kb-ebsco-java.postman_collection.json

```diff
@@ -8757,6 +8758,295 @@
 										}
 									},
 									"response": []
+								},
+								{
+									"name": "with valid packageId and filtered by tags",
+									"event": [
```

Comment:
Really minor comment -- on the naming of these -- could we add in the (including resources) -- test file is large may help in finding/identifying tests, thx -- for example
`with valid packageId including resources filtered by tags`

Reply:
Thanks Carole. It really makes sense. I will rename current tests in the next PR related api tests